src/components/icons/ArrowDownIcon/ArrowDownIcon.tsx

```diff
@@ -0,0 +1,19 @@
+import * as classNames from 'classnames';
+import * as React from 'react';
+import Icon, { IconProps } from '../Icon';
+import styles from './styles.module.scss';
+
+const ArrowDownIcon: React.FC<IconProps> = ({ color = 'primary', className, ...props }) => {
```

Comment:
У тебя тип для color это енам, соответственно ты можешь передать в этот проп только значение из енама.
По идее тебе линтер должен ошибки выводить во всех таких местах. Надо пройти по всему проекту и поправить везде, где вместо значения из какого-либо енама (не только цвет) передается строка
```suggestion
const ArrowDownIcon: React.FC<IconProps> = ({ color = ColorType.primary, className, ...props }) => {
```

Reply:
done